### PR TITLE
Behat test to confirm that re-enabling the CI build, creates a new CI build item.

### PR DESCRIPTION
**Pull request by HelenaEksler**
#156

Follows up #158.

### CODE REVIEW

File: behat/features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -462,4 +462,78 @@ protected function getNodeByTitleAndBundle($title, $bundle) {
...
+   *
+   * @param $node
+   *  The CI Build node.
+   * @param $status
+   *  Status field value. Defaults to "queue".
+   *
+   * @return array
+   *  Return array of messages IDs.
+   */
+  public function getCiBuildItemsByStatus($node, $status ='queue') {
```
Comment (HelenaEksler) [inline behat/features/bootstrap/FeatureContext.php]: @amitaibu Is there a way to re-use the function from the `shoov_ci_build` module in behat and not to duplicate the code?

Comment (amitaibu) [inline behat/features/bootstrap/FeatureContext.php]: which function? Can't you just call it?

Comment (HelenaEksler) [inline behat/features/bootstrap/FeatureContext.php]: https://github.com/shoov/shoov/blob/master/shoov/modules/custom/shoov_ci_build/shoov_ci_build.module#L446-L460

I tried just to use it - but it didn't work...

Comment (amitaibu) [inline behat/features/bootstrap/FeatureContext.php]: it is _supposed_ to work. You'll have to check why not..

**Comment by HelenaEksler** [comment]
@amitaibu Ready for review.

File: behat/features/ci_build.feature [hunk 2]
```diff
@@ -14,3 +14,16 @@ Feature: CI Build
     | Gizra/Gizra             |
     | drupal/drupal           |
     | DavidKohav/test-example |
+
+  @api
+  Scenario: Check after disabling of the CI build a new CI build item is removed.
+    Given I login with user "admin"
```
Comment (amitaibu) [inline behat/features/ci_build.feature]: tests shouldn't not be with admin - can we create a new group and do each test on different group?

**Comment by HelenaEksler** [comment]
selection_078

@amitaibu Like this?

**Comment by amitaibu** [comment]
:+1:

**Comment by HelenaEksler** [comment]
@amitaibu Ready.

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**